### PR TITLE
scroll to currently playing when switching to up-next tab and dont sc…

### LINKED ISSUE #160: Scroll to the playing video when switching from the Related tab to the Up Next tab
Having to scroll down a long list of videos every time you want to see what's up next after checking out related videos is frustrating.

### PR DESCRIPTION
**Pull request by SirZach**
…roll to bottom when adding a song in the related tab

closes #160

### CODE REVIEW

File: assets/js/components/playlist.js [hunk 1]
```diff
@@ -52,8 +52,19 @@ function PlaylistController($rootScope, $scope, $video, $storage, $log, $notific
...
+
+    $list.animate({
+      scrollTop: $playing.prop('scrollHeight')
+    }, 1000);
+  }
+
   this.setTab = function(tab) {
     this.activeTab = tab;
+
+    window.setTimeout(() => this.scrollToCurrentlyPlaying(), 250);
```
Comment (SirZach) [inline assets/js/components/playlist.js]: when you click the tab, the new set of playlistitems don't show up right away. i tried running a `$scope.digest()` to let angular handle the binding update but it complained that the digest loop was already happening so...i'm like whatevaaaa

**Comment by TheConnMan** [comment]
When I switch back to the playlist from the related tab the list scrolls a little, but doesn't scroll to the bottom of the currently playing.

**Comment by SirZach** [comment]
can you create a gif or something? it scrolled currently playing in view for me

**Comment by TheConnMan** [comment]
scroll-to-bottom

**Comment by SirZach** [comment]
thanks for the gif. updated to use native dom javascript that i never knew existed. works much better

**Comment by SirZach** [comment]
also closes #60 now

**Pull request merged**